qiskit/compiler/assemble.py

```diff
@@ -34,6 +34,7 @@ def assemble(experiments,
              qubit_lo_range=None, meas_lo_range=None,
              schedule_los=None, meas_level=2, meas_return='avg', meas_map=None,
              memory_slot_size=100, rep_time=None, parameter_binds=None,
+             parametric_pulses=None,
```

Comment:
why user can input this?

Reply:
This just follows the format for `assemble`. I was uncertain, too, but there are other parameters there that a user doesn't _truly_ have control over if they play it on a backend. I can see it being useful, though, if you decide you want a Qobj with pulses described this way. Otherwise, you have to modify the backend to overwrite it, which I don't think is the way Ali would have it done.